php-checks/src/test/resources/checks/SSLCertificatesVerificationDisabledCheck.php

```diff
@@ -1,26 +1,21 @@
 <?php
 
-$verify_host_on = '2';
-curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, $verify_host_on);
-curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, '2');
-curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, 2);
-other_func($curl, CURLOPT_SSL_VERIFYHOST, 0);
-
 $verify_peer_on = TRUE;
 curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, $verify_peer_on);
 curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, true);
 curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, '1');
 curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, 1);
+curl_setopt($curl, CURLOPT_SSL_VERIFYPEER); // OK - is enabled by default
+
 other_func($curl, CURLOPT_SSL_VERIFYPEER, 0);
 curl_setopt($curl, CURLOPT_OTHER_KEY, 0);
 
-$verify_host_off = FALSE;
-curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, $verify_host_off); // Noncompliant {{Activate SSL/TLS certificates chain of trust verification.}}
-curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, TRUE); // Noncompliant
-curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, '0'); // Noncompliant
-curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, 0); // Noncompliant
-
 $verify_peer_off = '0';
-curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, FALSE); // Noncompliant
+curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, FALSE); // Noncompliant {{Enable server certificate validation on this SSL/TLS connection.}}
+                                         //^^^^^
 CURL_setopt($curl, CURLOPT_SSL_VERIFYPEER, $verify_peer_off); // Noncompliant
+curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, '0'); // Noncompliant
```

Comment:
We raise issues on the following lines:
```
curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, "yes");
curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, "sure!");
```
I think that would be FPs (which were already present in the existing implementation of the rule).
It's probably not a big deal, and we could include them in our tests.